README.md

```diff
@@ -1 +1,12 @@
-# project-1
\ No newline at end of file
+# TaskMaster
+
+## What is TaskMaster
+Taskmaster is a modern, web-based task management application designed to help users stay organized, productive, and focused. Whether you're managing your personal to-do list or tackling complex projects, Taskmaster offers an intuitive, feature-rich platform to streamline your task management process. With a clean interface and powerful functionality, Taskmaster helps you organize, prioritize, track, and complete your tasks efficiently.
+
+
+## Features
+- Task Creation: Add tasks with a title, description, due date, and priority. Include tags or categories to keep your tasks organized.
+- Mark tasks with priority labels by changing the colors to focus on the most important items first.
+- Responsive Design: Optimized for desktop use, so you can manage tasks with ease and efficiency.
+
```

Comment:
I only saw the notification on discord to commit and didnt to in to see the full thing. once you guys approve ill just add it here at the end.